packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -62,7 +65,7 @@ export const getStyles = memoizeFunction((
       borderWidth: '1px',
       borderStyle: 'solid',
       borderColor: checkboxBorderColor,
-      marginRight: '8px',
+      margin: '0 4px',

```

Comment:
yes there is, for sure. However, it's better to just not rely on it. 

For example, there is a "far" mode that the Planner team added, which renders the checkbox at the end. When rendered in this mode, your margin should actually be applied to the left, even though we are still in LTR.

If you always use symmetric margins, you never have these sorts of cases. Another example; if you leave the label off, you end up having no padding with my changes. The label has -4px, the box has 4px, equaling 0 padding.